### DISCUSSION TITLE
关于新版本CDN

### DISCUSSION THREAD

**Discussion by Sprit-Fun** [mirai-mamori/Sakurairo#511] [Q&A]
请问一下新版本现在需要提供API请求资源request，但是我的root和gallery manifest并没有这些资源，导致图片无法加载（例如bilibili.png, github.png）。请问是需要将资源放入root来加载吗？谢谢！

**Comment by Sprit-Fun** [comment]
![image](https://user-images.githubusercontent.com/43423211/166134885-09318c27-52d1-48a8-8912-75e34926a6d2.png)
![image](https://user-images.githubusercontent.com/43423211/166134893-19639da1-dfdc-4d84-aaa3-d3b1f186ec8d.png)
![image](https://user-images.githubusercontent.com/43423211/166134908-90815acf-db44-481e-a165-d8956298db33.png)

更新：应该是请求都在root里面的资源，但是root没有相关资源。我找了一遍设置，好像没找到自定义CDN资源地址，请问是否应该将资源放入root？

**Comment by Sprit-Fun** [comment]
![image](https://user-images.githubusercontent.com/43423211/166135140-185923e9-0be1-4dc3-91ee-9678898218e1.png)
![image](https://user-images.githubusercontent.com/43423211/166135161-5b414886-4218-4c2c-b118-eaee01cf3b91.png)

发现好像是因为CDN SSL/ CORS请求被block了？不知道是不是这个原因。

**Comment by Sprit-Fun** [comment]
已经清除了所有缓存，貌似还是无法收到response。是否与CDN host有关？使用了其他end user进行请求也是无法收到response。

**Comment by Sprit-Fun** [comment]
经过测试，公共vpn可以访问：
https://jscdn-x.panbaidu.cn/source/Sakurairo_Vision/@2.4/display_icon/sora/bilibili.png
但是不知原因我的CDN请求不一样，导致无法获得资源：
https://x.jscdn.host/release/ucode-x/source/Sakurairo_Vision/@2.4/display_icon/flat_colorful/bilibili.png

**Comment by Sprit-Fun** [comment]
![image](https://user-images.githubusercontent.com/43423211/166135876-9c92259d-7077-4350-ada9-6febb117f4db.png)
设置了正确CDN好像还是请求了错误CDN：
![image](https://user-images.githubusercontent.com/43423211/166135886-53e50e7c-80c4-44d8-b642-f37544df7479.png)

**Comment by Sprit-Fun** [comment]
发现x.jscdn貌似是很多默认请求CDN：
![image](https://user-images.githubusercontent.com/43423211/166136014-50b9a216-d77b-4d60-a094-c069ac631702.png)
请问要怎么设置到jscdn.x来获取正确资源？尝试了不同设置最后都还是发送request到x.jscdn。

**Comment by Sprit-Fun** [comment]
发现与 #499  问题描述一致（重复），但是更换视觉资源路径貌似还是走了这个CDN

**Reply by CN-GuoZiyang** [reply]
我这现在也是这样，更换视觉资源路径，但是很多资源不走自定义路径，建议提个 issue

**Reply by Sprit-Fun** [reply]
> 我这现在也是这样，更换视觉资源路径，但是很多资源不走自定义路径，建议提个 issue

有人提了，应该下个版本会修复
